test/evp_extra_test.c

```diff
@@ -2414,6 +2414,104 @@ static int test_evp_iv(int idx)
         EVP_CIPHER_free((EVP_CIPHER *)type);
     return ret;
 }
+static int test_ecpub(int idx)
+{
+    int ret = 0, len, savelen;
+    int nid;
+    unsigned char buf[1024];
+    unsigned char *p;
+    const unsigned char *q;
+    EVP_PKEY *pkey = NULL, *pkey2 = NULL, *pkeytmp = NULL;
+    EVP_PKEY_CTX *ctx = NULL;
+    EC_KEY *ec = NULL;
+
+    switch(idx) {
```

Comment:
Why not put these NIDs into an array and use `nid = array[idx]`?  With a switch to `OSSL_NELEM` in the ADD_ALL_TESTS macro later.

Reply:
No reason at all...fixup pushed, and thanks for the suggestion